xdis/op_imports.py

```diff
@@ -181,8 +215,8 @@ def get_opcode_module(version_info=None, variant=None):
             pass
     else:
         vers_str += variant
-
-    return op_imports[canonic_python_version[vers_str]]
+    v = get_target_version(vers_str)
```

Comment:
I am not a big fan of doing this kind of thing - approximate opcode versioning.  In the past, close version numbers have different opcodes. 

That said, it would be okay to do provided this has been _explicitly_ asked which would man another parameter to `get_opcode_module` which allows fuzziness, the default being not to allow fuzziness.

Reply:
i do this is for my action throw a error ,  to which i set a python version `python-lastest`(3.10.5).
https://github.com/serfend/pydumpck/runs/6978042434?check_suite_focus=true#step:6:796
```bash
------------------------------- Captured stdout --------------------------------
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
I don't know about Python version '3.10.5' yet.
Python versions 3.9 and greater are not supported.
=========================== short test summary info ============================
ERROR tests/test_array_extensions.py - KeyError: '3.10.5'
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.68s ===============================
```

Reply:
Version 3.10.5 is noted in the current master. And this is what I prefer, when a new version comes out it is tested and noted in the code. 

I you want to improve the error message to be something more user friendly that is fine. 

If you are writing an application that uses xdis you can add that difference approximation in your code. That is your decision if you want this kind of flakiness. If however you are using code like one of my decompilers then again, I am not fond of this kind of approximate behavior.  A user has to explicitly request flakiness.

Reply:
oh,get.